src/lib.rs

```diff
@@ -135,8 +141,9 @@ const DEFAULT_BURST_SIZE: u32 = 8;
 /// ```
 #[derive(Clone, PartialEq, Eq, Debug)]
 pub struct GovernorConfigBuilder {
-    period: Duration,
+    period:     Duration,
```

Comment:
Indentation looks odd here. Maybe run `cargo fmt`?

Reply:
Ah yeah that's my own doing. I tend to aline them all on 1 line. Will fix